src/openrct2-ui/windows/Sign.cpp

```diff
@@ -323,6 +323,9 @@ static void window_sign_paint(rct_window* w, rct_drawpixelinfo* dpi)
 static void window_sign_viewport_rotate(rct_window* w)
 {
     rct_viewport* view = w->viewport;
+    if (view == nullptr)
+        return;
+
     w->viewport = nullptr;
 
     view->width = 0;
```

Comment:
I assume it crashed here. Why is this needed? Can't this just be removed, or moved down to the other null check?

Reply:
Yeah, I think this is the correct fix:
```c
if (w->viewport != nullptr)
{
    w->viewport->width = 0;
    w->viewport = nullptr;
}
```

Reply:
I applied @IntelOrca suggestion.